core/trading_engine.py

```
@@ -673,61 +960,164 @@ async def _execute_signals(self, entry_signals: List[Signal], exit_signals: List
+                                    order_status = connector.get_order_status(signal.symbol, result.order_id)
+                                    if order_status and order_status.status == OrderStatus.FILLED:
+                                        result = order_status  # 체결된 주문으로 업데이트
+                                        logger.info(f"✅ 주문 체결 확인: {signal.symbol} (폴링 {i+1}회)")
+                                        break
+                                    elif order_status and order_status.status in [OrderStatus.CANCELLED, OrderStatus.REJECTED]:
+                                        logger.error(f"❌ 주문 취소/거부됨: {order_status.status.value}")
+                                        return
+                                else:
+                                    logger.warning(f"⚠️ {exchange_name}: get_order_status 메서드 없음, 폴링 중단")
+                                    break
+                            except Exception as poll_error:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)